### DISCUSSION TITLE
Version planning

### DISCUSSION THREAD

**Discussion by maxpowel** [patrykandpatrick/vico#586] [Questions] [answered]
Hello,
There is version 2 in alpha but version 1 is still getting new versions and I'm wondering what is the plan.
 * Will v1 still get new features or the main effort will be for v2?
 * Should I start using v2?

Any information about your plans will be great!

Thanks!

**Comment by patrickmichalik** [comment, marked as answer]
Hello! We’re focusing primarily on Vico 2. Vico 1 will receive no new features, only bug fixes, and it will lose support a year after Vico 2 is made stable or later.

Vico 2 includes features and enhancements you won’t find in Vico 1. However, in the alpha stage, API changes are to be expected. Please take a look at the release notes for examples. Whether you should update now depends on if you need these features and enhancements and if you’re ready to adapt to API changes.

**Comment by maxpowel** [comment]
Thank you so much, now it is pretty clear!

**Reply by patrickmichalik** [reply]
You’re welcome!
